libs/cardano-ledger-core/src/Cardano/Ledger/Core.hs

```diff
@@ -235,6 +235,11 @@ class
   getTotalRefundsTxBody pp lookupStakingDeposit lookupDRepDeposit txBody =
     getTotalRefundsTxCerts pp lookupStakingDeposit lookupDRepDeposit (txBody ^. certsTxBodyL)
 
+  -- | This function is not used in the ledger rules. It is only used by the downstream
+  -- tooling to figure out how many witnesses should be supplied for Genesis keys.
+  genesisKeyHashCount :: TxBody era -> Int
+  genesisKeyHashCount _ = 0
```

Comment:
Why have a default implementation - is it because they don't exist starting with Conway?

Reply:
Precisely. All future eras after Babbage will have it set to 0, therefore it is a safe default.